turborepo-tests/integration/tests/filter-run.t

```diff
@@ -26,4 +26,18 @@ Setup
    Tasks:    0 successful, 0 total
   Cached:    0 cached, 0 total
     Time:\s*[\.0-9]+m?s  (re)
-  
\ No newline at end of file
+  
+
```

Comment:
This works as an integration test, but we have pretty good fixtures in the `filter.rs` unit tests and feels like we could get away with just those.

Reply:
From a quick scan, I'm not sure we have unit tests that specifically handle git ranges where you can specify the packages changed. TestChangeDetector works by storing the changed packages for a git ref range, not the changed files.